src/mudlet.cpp

```diff
@@ -80,6 +81,20 @@
 #include <QVariantHash>
 #include <QRandomGenerator>
 #include <zip.h>
+#include <QStyle>
+#ifdef Q_OS_LINUX
+#elif defined(Q_OS_WIN32)
+#include <QSettings>
+#endif
+
+#if defined(Q_OS_MAC)
+// wrap in namespace since `Collection` defined in these headers will clash with Boost
+namespace coreMacOS {
+#include <CoreFoundation/CoreFoundation.h>
+#include <CoreServices/CoreServices.h>
+}
+#endif
```

Comment:
The branch for Linux is not needed - it just makes things a tad more complicated:
```diff
-#ifdef Q_OS_LINUX
-#elif defined(Q_OS_WIN32)
+lif defined(Q_OS_WIN32)
 #include <QSettings>
-#endif
-
-#if defined(Q_OS_MAC)
+#elif defined(Q_OS_MAC)
 // wrap in namespace since `Collection` defined in these headers will clash with Boost
 namespace coreMacOS {
 #include <CoreFoundation/CoreFoundation.h>
 #include <CoreServices/CoreServices.h>
 }
 #endif
```